lib/glific/flows/action.ex

```diff
@@ -518,6 +519,15 @@ defmodule Glific.Flows.Action do
       Flows.start_contact_flow(flow.id, contact, %{"parent" => context.results})
     end)
 
+    action.groups
+    |> Enum.each(fn group ->
+      group = Repo.get_by(Group, %{id: group["uuid"]})
+
+      Task.async(fn ->
```

Comment:
Do you not want to wait for this task to finish and then run the next iteration?

Reply:
I'm just using async to calling the function that will start flow for collection rest the flow will be queued for the collection and will take some time to start flow for all

Reply:
ah okay